src/App.vue

```diff
@@ -450,9 +450,9 @@ export default class App extends Mixins(
 
     // now that LaunchDarkly has been updated (ie, in case of user targeting),
     // check whether to use this Entity Dashboard or the new Business Dashboard
-    if (GetFeatureFlag('use-business-dashboard')) {
+    if (GetFeatureFlag('use-business-dashboard') && (this.$route.name === Routes.DASHBOARD)) {
```

Comment:
This was behaving incorrectly when the (new) Business Dashboard UI redirected to this (old) Entity Dashboard to perform a local filing (eg, COD).